MAUI/DataGrid/export-to-excel.md

```diff
@@ -0,0 +1,548 @@
+
+{% tabs %}
+{% highlight c# %}
+DataGridExcelExportingOption option = new DataGridExcelExportingOption();
+option.BottomTableSummaryStyle = new DataGridExportCellStyle()
+{
+    BackgroundColor = Color.FromArgb("0074E3"),
+    BorderColor = Colors.DarkBlue,
+    TextColor = Colors.White,
+    FontInfo = new DataGridExportFontInfo()
+    {
+        Bold = true,
+    }
+};
+{% endhighlight %}
+{% endtabs %}
+
+<img alt="Export DataGrid to Excel format with style applied for table summary rows at the bottom" src="Images\Export-to-Excel\maui-datagrid-bottom-table-summary-style.png" width="689"/> 
+
+#### GroupCaptionStyle
+
+The SfDataGrid supports exporting the GroupCaptionSummaries with custom style by using the `DataGridExcelExportingOption.GroupCaptionStyle` property.
+
+{% tabs %}
+{% highlight c# %}
+DataGridExcelExportingOption option = new DataGridExcelExportingOption();
+option.GroupCaptionStyle = new DataGridExportCellStyle()
+{
+    BackgroundColor = Color.FromArgb("0074E3"),
+    BorderColor = Colors.DarkBlue,
+    TextColor = Colors.White,
+    FontInfo = new DataGridExportFontInfo()
+    {
+        Bold = true,
+    }
+};
+{% endhighlight %}
+{% endtabs %}
+
+<img alt="Export DataGrid to Excel format with style applied for Group summary rows" src="Images\Export-to-Excel\maui-datagrid-group-caption-style.png" width="689"/> 
+
+#### HeaderStyle
+
+The SfDataGrid allows exporting the column headers with custom style by using the `DataGridExcelExportingOption.HeaderStyle` property.
+
+{% tabs %}
+{% highlight c# %}
+DataGridExcelExportingOption option = new DataGridExcelExportingOption();
+option.HeaderStyle = new DataGridExportCellStyle()
+{
+    BackgroundColor = Color.FromArgb("0074E3"),
+    BorderColor = Colors.DarkBlue,
+    TextColor = Colors.White,
+};
+{% endhighlight %}
+{% endtabs %}
+
+<img alt="Export DataGrid to Excel format with styling for header cells" src="Images\Export-to-Excel\maui-datagrid-header-style.png" width="689"/> 
+
+#### RecordStyle
+
+The SfDataGrid allows exporting the records with custom style by using the `DataGridExcelExportingOption.RecordStyle` property.
+
+{% tabs %}
+{% highlight c# %}
+DataGridExcelExportingOption option = new DataGridExcelExportingOption();
+option.RecordStyle = new DataGridExportCellStyle()
+{
+    BackgroundColor = Color.FromArgb("#AFD5FB"),
+    BorderColor = Colors.DarkBlue,
+    TextColor = Colors.Black,
+    FontInfo = new DataGridExportFontInfo()
+    {
+        Italic = true,
+    }
+};
+{% endhighlight %}
+{% endtabs %}
+
+<img alt="Export DataGrid to Excel format with styling for record cells" src="Images\Export-to-Excel\maui-datagrid-record-style.png" width="689"/>
+
+#### GroupSummaryStyle 
+
+`SfDataGrid` supports exporting the `GroupSummary` rows with custom style by using the `DataGridExcelExportingOption.GroupSummaryStyle` property.
```

Comment:
The `SfDataGrid` allows exporting the group summary rows with custom style by using the `DataGridExcelExportingOption.GroupSummaryStyle` property.